CHANGELOG.md

```diff
@@ -123,6 +123,9 @@
 ### Tools
 
 * [CHANGE] tsdb-index: Rename tool to tsdb-series. #6317
+* [CHANGE] copyblocks: The flags for copyblocks have been changed to align more closely with other tools. #6607
```

Comment:
Please don't forget to rebase and move these entries to `main` again.

Reply:
:+1: I'll be sure to do this before I do the last round of manual testing for the new changes. I marked this as draft until that's completed.